packages/web-components/fast-components/src/slider/slider.template.ts

```diff
@@ -1,10 +1,13 @@
 import { html } from "@microsoft/fast-element";
 import { ref } from "@microsoft/fast-element";
+import { Orientation } from "@microsoft/fast-web-utilities";
 import { Slider } from "./slider";
 
 export const SliderTemplate = html<Slider>`
     <template
         role="slider"
+        class="${x => (x.readOnly ? "readonly" : "")} 
+        ${x => (x.orientation === Orientation.horizontal ? "horizontal" : "vertical")}"
```

Comment:
It looks like if x.orientation is undefined it will be vertical. Is that what you want?

If not I think you can just do:

```ts
${x => x.orientation || Orientation.horizontal}
```

The `Orientation` enum uses strings so you should just need to use the enum.

Reply:
orientation is defaulted true, so if the element doesn't have the attribute specified it will always be horizontal. Good point on the string thing, doh! :)

Reply:
Do you mean defaulted to `Orientation.horizontal`?

Reply:
doubly done, thanks, looks much better